app/controllers/api/v2/instances_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class Api::V2::InstancesController < Api::V1::InstancesController
   def show
-    expires_in 3.minutes, public: true
+    cache_even_if_authenticated!
```

Comment:
Same as for V1, this includes an account, and thus user-dependent relationships, so it should probably only be cached for unauthenticated visitors.